dotnet/src/VectorDataIntegrationTests/VectorDataIntegrationTests/CRUD/RecordConformanceTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using VectorDataSpecificationTests.Models;
+using VectorDataSpecificationTests.Support;
+using VectorDataSpecificationTests.Xunit;
+using Xunit;
+
+namespace VectorDataSpecificationTests.CRUD;
+
+public class RecordConformanceTests<TKey>(VectorStoreFixture fixture)
+    : ConformanceTestsBase<TKey, SimpleModel<TKey>>(fixture) where TKey : notnull
+{
+    [ConditionalFact]
+    public async Task ReadingAndDeletingNonExistingRecordDoesNotThrow()
```

Comment:
I'd prefer a bit more separate where each test exercises one thing, if possible (this one exercises both Get and Delete, the one below even more things).